boost-common/src/main/java/io/openliberty/boost/common/config/BoostProperties.java

```diff
@@ -29,6 +29,8 @@
     public static final String DATASOURCE_PASSWORD = "boost.db.password";
     public static final String DATASOURCE_CREATE_DATABASE = "boost.db.createDatabase";
 
+    public static final String PASSWORD_ENCRYPTION_KEY = "boost.db.password.key";
+
```

Comment:
This looks like it is being used to encrypt all passwords (presuming there will be properties for other boosters that require passwords). At line 228 in LibertyServerConfigGenerator, this constant is used to encrypt all password properties. So I question the value of "boost.db.password.key"...should it just be "boost.password.encryption.key"?

Reply:
Also - we are keying off of the "boost.db." prefix to auto generate variables on the datasource property elements in server.xml. If we do decide to have this password key specific to the datasource password, we would need to change the naming scheme, or make sure we don't generate a variable called "password.key".

Reply:
I added an encryption type value along with the properties we want to encrypt in BoostProperties.

I also changed the name of the aes key property to `boost.aes.key`. It will be used for all of the aes encoding we do.